index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8" />
+    <title>Homepage</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="home.css">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="My Logo">
+        <img src="image.png" alt="">
+    </div>
+    <div class="content">
+        <div class="topic">
+            <h1>I’m Bantealem
+                Glad to see you!</h1>
+        </div>
+        <div class="text">
+            <h4>
+                I’m a software developer!
+                I can help you build a product ,
+                feature or website Look through some of my work and
+                experience!
+                If you like what you see and have a project you need coded,
+                don’t hestiate to contact me.
+            </h4>
```

Comment:
- [ ] Kindly change your tag from an `h4` to a `p` tag as this is not a heading but a paragraph. 🙏